packages/plugin-svgo/src/config.js

```diff
@@ -32,14 +36,18 @@ function extractPlugins(config) {
 }
 
 function mergePlugins(configs) {
-  const plugins = configs.reduce(
-    (merged, config) => deepmerge.all([merged, ...extractPlugins(config)]),
-    {},
-  )
-  return Object.keys(plugins).reduce((array, key) => {
-    array.push({ [key]: plugins[key] })
-    return array
-  }, [])
+  return configs
+      .flatMap(extractPlugins)
```

Comment:
Node 10 doesn't seem to support flatMap
Consider changing this like to:
.reduce((configsArr,config)=>configArr.concat(extractPlugins(config)),[])

Refference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap

Else, you would need to remove compatibility to Node 10.

Reply:
> Else, you would need to remove compatibility to Node 10.

[You need to do this anyway](https://github.com/svg/svgo/issues/1308#issuecomment-822522625):

> _So while the 2.0 release in Feb 2021 mentions needing `10.13.x` or newer, in this case to resolve this issue (as it appears to be NodeJS related not specifically `svgo`, you need NodeJS `11.x.x` or newer_

Otherwise users may risk breakage as the linked issue describes and my investigation revealed was due to issues with NodeJS 10.